examples/pytorch/preprocessing_dist_training/metis_creation.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+import os
+import glob
+import json
+import argparse
+from collections import defaultdict
```

Comment:
my understanding is that this script converts a graph in certain format to the format required by ParMETIS?
in our example, we convert an OGB graph into the ParMETIS format. I guess every user may have their own original format?